kit/inputs/src/UIKeyTextView.tsx

```diff
@@ -123,9 +125,11 @@ export const UIKeyTextView = React.forwardRef<UIMaterialTextViewRef, UIKeyTextVi
                 onBlur={onBlur}
                 onChangeText={onChangeText}
                 onKeyPress={onKeyPress}
+                onDone={onDone}
```

Comment:
I don't think it's a good idea to add an extra `onDone` prop inside of a pure `UIMaterialTextView` component. 
I'd suggest to add this ActionButton component inside of `UIKeyTextView` only

Reply:
@sertony But UIMaterialTextView have useMaterialTextViewChildren which is made especially for similar buttons, and thats why I pass onDone here

Reply:
I would say that sth is actually wrong with `useMaterialTextViewChildren`, it shouldn't actually have `onDone` argument, it rather should be passed as child component, as it was supposed to be. Solve bigger amount of cases > solve one case.

Reply:
@wearedusa I've checked the code of UIMaterialTextView. You can pass the `.Icon` element as a children of the input, as it's done here: https://github.com/tonlabs/UIKit/blob/676a56bf3759edebab71b2e76486f6ad9b108cc6/casts/addressText/src/UIAddressTextView.tsx#L155..L162

Hence, you won't need any unwanted `onDone` props and even ActionButton ;)